client/src/mapDetails.js

```diff
@@ -0,0 +1,5 @@
+let myMap = L.map('my-map').setView([39.73, -104.99], 15)
+L.titleLayer('',{
+    maxZoom:17,
+    attribution:'',
+}).addTo(myMap)
```

Comment:
This is another file that appears to not be used, and if it's not being used we can remove it to cut down on some bloat.